notebooks/tfx_pipelines/pipeline/solutions/tfx_pipeline_vertex.ipynb

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
<p>Nit: consider calling out the services used in the learning objectives.</p><p> </p><ol><li>Containerize your TFX code into a pipeline package using Cloud Build.</li><li>Use the TFX CLI to compile a TFX pipeline.</li><li>Deploy a TFX pipeline version to run on Vertex Pipelines using the Vertex Python SDK.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done!</p>